rdrf/rdrf/templates/rdrf_cdes/alert_language_is_not_supported.html

```diff
@@ -0,0 +1,34 @@
+{% load i18n %}
+{% load get_language_settings_codes %}
+{% get_language_settings_codes as LANGUAGES %}
+
+<script type="text/javascript">
+    $(document).ready(function(){
+        const browser_language = navigator.language;
+        const supported_languages = {{ LANGUAGES|safe }};
+        const is_browser_language_in_settings = [browser_language, browser_language.split('-')[0]].some((language) => supported_languages.includes(language));
+        const existing_not_support_language = localStorage.getItem('not_support_language');
```

Comment:
I'm not sure I understand the purpose of the `not_support_language`

Reply:
There might be a case that a user selects browser language as `Japanese` firstly, and the alert message shows '_Japanese is not currently supported_', however, after this alert is dismissed, the user switch browser language to `Korean`, which is also not supported. In this case, I think we should again popup an alert telling the user language '_Korean_' is not supported.